awscli/examples/networkflowmonitor/list-tags-for-resource.rst

```diff
@@ -0,0 +1,15 @@
+**To list the tags**
+
+The following ``list-tags-for-resource`` example returns all the tags associated with the specified resource. ::
+
+    aws networkflowmonitor list-tags-for-resource \
+        --resource-arn arn:aws:networkflowmonitor:us-east-1:123456789012:monitor/Demo
+
+Output::
+
+    {
+        "tags": {
+            "Value": "Production",
+            "Key": "stack"
+        }
+    }
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link